requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Specify libhtp and suricata-update requirements.
+#
+# Format:
+#
+#   name {repo} {branch|tag}
+libhtp https://github.com/OISF/libhtp 0.5.x
```

Comment:
should this be 0.5.40?

Reply:
So leave as-is.  Updating will become part of the versioning process I guess.  So its pinned to a tag in a Suricata tag.

Reply:
never really looked into how, but for libhtp we specify the version only here https://github.com/OISF/libhtp/blob/0.5.x/VERSION

Reply:
> can we use this file as the source of the htp version check in configure?

We can certainly use some `sed` to extract it. But how do the version check if its simply `0.5.x`? Just allow any libhtp version that matches 0.5.?

Reply:
possibly something like what we have here? https://github.com/OISF/suricata/blob/master/configure.ac#L1526